src/components/displayDocuments/displayDocuments.tsx

```diff
@@ -8,10 +8,10 @@ const SectionContainer = styled.section`
   background-color: #e5e5e5;
 `;
 
-export const DisplayDocuments: React.FunctionComponent = () => {
-  const displayCards = (): JSX.Element[] => {
+export const DisplayDocuments: React.FunctionComponent = (): ReactElement => {
```

Comment:
Think this is sufficient as FunctionComponent is already declared.

```suggestion
export const DisplayDocuments: React.FunctionComponent = () => {
```